SKILL.tsv

```diff
@@ -4460,9 +4460,9 @@ SKILL_20150714_004459	$When you attack an enemy affected by [Shrink Body] while
 SKILL_20150714_004460	$Enhance Left Arm: Enlarge Size Speciality
 SKILL_20150714_004461	{memo X}When you attack an enemy that is in [Swell Body] when [Left Arm Enhancement] is being activated, the weakened ATK rate status will continue. Duration will increase by 8 secs per Attribute level and the weakened ATK rate can be overlapped max 10 times.
 SKILL_20150714_004462	$Right Arm Enhance: Swiftness
-SKILL_20150714_004463	If you are not equipped with a Sub Weapon,
+SKILL_20150714_004463	$Not using a dagger nor a shield as an offhand gives you a 6% higher accuracy and evasion rate per attribute level.
```

Comment:
higher accuracy and evasion rate per attribute level -> higher accuracy and evasion per attribute level